codalab/apps/web/templates/web/competitions/upload_s3_competition.html

```diff
@@ -0,0 +1,94 @@
+{% extends 'base.html' %}
+
+{% block head_title %}Create Competition{% endblock head_title %}
+{% block page_title %}Create Competition{% endblock page_title %}
+
+{% block content %}
+    <div id="create_competition" class="panel panel-default s3-panel">
+        <div class="panel-body">
+            <div class="row">
+              <h3 class="center-block"  style="width:80%;">
+                To create a competition, upload a bundle with the definition of the new competition.
+                <a href="https://github.com/codalab/codalab/wiki/User_Competition-Roadmap"
+                   title="Get help with creating a competition" target="_blank">
+                    <span class="glyphicon glyphicon-question-sign"></span>
+                </a>
+              </h4>
+            </div>
+
+            <form id="s3_upload_form" action="" method="post">
+                {% csrf_token %}
+                <div class="form-group">
+                  <button type="button" id="submit-button" class="btn btn-primary">Upload Competition Bundle</button>
+                </div>
+                <div class="row">
+                  <div class="form-group center-block" style="width:50%; height: 10px;">
+                    {{ form }}
```

Comment:
This is an informative bit of User Interface for people to see the filename of what they've uploaded.